app/authconfig_data.go

```diff
@@ -20,7 +21,15 @@ func addAuthConfigs(management *config.ManagementContext) error {
 		return err
 	}
 
-	return addAuthConfig(localprovider.Name, client.LocalConfigType, true, management)
+	if err := addAuthConfig(ping.Name, client.PingSamlConfigType, false, management); err != nil {
+		return err
+	}
+
+	if err := addAuthConfig(localprovider.Name, client.LocalConfigType, true, management); err != nil {
+		return err
+	}
+
+	return addAuthConfig("authconfig", client.AuthConfigType, false, management)
```

Comment:
why did you add this line? I'd expect this file to _only_ be adding ping, nothing else.